jetstream/lifestyles-images-onboarding-experiment-v3.toml

```diff
@@ -0,0 +1,39 @@
+## EXPERIMENT SPECIFIC
+[experiment]
+segments = ['attributed', 'non_attributed']
+
+start_date = "2023-06-14"
+
+enrollment_period = 14
+
+end_date = "2023-07-26"
+
+#reference_branch = "control"
+
+
+
+## SEGMENTS
+# A segment for users that were acquired through ctd creatives
+# first run feature.
+[segments]
+
+[segments.attributed]
+select_expression = 'max(eligible_segment = TRUE)'
```

Comment:
You might want to do `COALESCE(MAX(eligible_segment = TRUE), FALSE)` here to protect against nulls. If a user doesn't report a first session ping, they might have a null value here and this explicitly keeps those users out of this segment.